# Key code as defined in linux/inupt.h
uint16 key_code

# Key name string as defined in evtest, see http://elinux.org/images/9/93/Evtest.c
string key_name

# 'True' if key was pressed, 'False' otherwise
bool key_pressed